Public/OAuth/Show-ZDOAuthToken.ps1

```diff
@@ -0,0 +1,40 @@
+<#
+.Synopsis
+   Returns truncated token based on token ID
+.DESCRIPTION
+   Long description
+.EXAMPLE
+   Example of how to use this cmdlet
+.EXAMPLE
+   Another example of how to use this cmdlet
+#>
+function Show-ZDOAuthToken
+{
+    [CmdletBinding()]
+    [OutputType([int])]
+    Param
+    (
+        # Param1 help description
+        [Parameter(Mandatory=$true,
+                   ValueFromPipelineByPropertyName=$true,
+                   Position=0)]
+        $TokenID
```

Comment:
We should probably make $Token a global variable, similar to how Set-ZDHeader is doing. That way you can set the default value for the parameter something like:

[string]$TokenID = $Token